RetroBar/Languages/català.xaml

```diff
@@ -4,6 +4,7 @@
 
     <s:String x:Key="retrobar_properties">Propietats del RetroBar</s:String>
     <s:String x:Key="taskbar_tab">Barra de tasques</s:String>
+    <s:String x:Key="advanced_tab">Avançats</s:String>
```

Comment:
I think we can use "Avançades", "Avançada", or "Opcions avançades" for this one: multiple options to choose or advanced settings.
```suggestion
    <s:String x:Key="advanced_tab">Avançades</s:String>
```
```suggestion
    <s:String x:Key="advanced_tab">Avançada</s:String>
```
```suggestion
    <s:String x:Key="advanced_tab">Opcions avançades</s:String>
```

Reply:
I just saw both "Avançats" and "Opcions avançades" in the `explorer.exe` strings from XP, so I think we can keep this change unless you change your mind...